source/fundamentals/time-series.txt

```diff
@@ -0,0 +1,134 @@
+=======================
+Time Series Collections
+=======================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn about **time series collections** in
+MongoDB, and how to interact with them in the MongoDB Go driver.
```

Comment:
issue (non-blocking): In the other drivers we're starting to use constants should the name of these ever change. Up to you if you want to include it here, but it might be nice to add it.